cli/src/semgrep/commands/scan.py

```diff
@@ -658,6 +651,12 @@ def scan_options(func: Callable) -> Callable:
     hidden=True
     # help="contact support@semgrep.com for more information on this"
 )
+@click.option(
```

Comment:
Even though it's beta, a help message might still be useful here---at least the same "contact support" language